src/presenters/gouvernancePresenter.ts

```diff
@@ -198,7 +199,7 @@ function formatPluriel(count: number): 's' | '' {
 }
 
 type ComiteViewModel = Readonly<{
-  dateProchainComite: string
+  date: string | null
```

Comment:
```suggestion
  date: string
```
Et au lieu de `null`, tu mets une chaine vide.